editor/animation_track_editor.cpp

```diff
@@ -4459,9 +4459,9 @@ AnimationTrackEditor::TrackIndices AnimationTrackEditor::_confirm_insert(InsertD
 			Array array;
 			array.resize(5);
 			array[0] = p_id.value;
-			array[1] = -0.25;
+			array[1] = MAX(-0.25, -animation->get_length() / 2.0);
```

Comment:
Not critical but I'd factor this out as:
```cpp
const float max_width = animation->get_length() / 2.0;
```
to avoid computing it twice.

Reply:
Good point. Done/